doc/RedisSessionHandler.md

```diff
@@ -4,16 +4,16 @@ This feature has been backported from Symfony 4. The `RedisSessionHandler` allow
 Last revision: https://github.com/symfony/symfony/commit/239a022cc01cca52c3f6ddde3231199369cf34c2
 
 ## Requirements
-- Symfony 3.x _(implies support for PHP >= 5.6, the feature is native as of Symfony 4)_
-- Redis extension or Predis _(using one of the following: RedisArray / RedisCluster / Predis\Client / RedisProxy / RedisClusterProxy)_
+- Symfony 3.x _(the feature is native as of Symfony 4)_
+- Redis extension _or_ Predis
 
 ## Configuration
 After installing the bundle, you have to configure proper services on your own to be able to use RedisSessionHandler.
 
 At first, you have to define service for Redis connection. It can be done in `app/config/services.yml`. Configuration should look like the following:
 ```yaml
     redis_session_handler_connection:
-        class: 'Redis' # RedisArray/RedisCluster/Predis\Client/RedisProxy/RedisClusterProxy are also supported
+        class: 'Redis' # Or one of: RedisArray, RedisCluster, Predis\Client, or  RedisProxy.
```

Comment:
Note, dropped RedisClusterProxy here as it's Symfony 4 feature, but didn't adapt the class for this as it's better if it stay as close to upstream as possible.